easybuild/easyconfigs/m/MXNet/MXNet-1.9.1-foss-2022a.eb

```diff
@@ -0,0 +1,71 @@
+name = 'MXNet'
+version = '1.9.1'
+
+homepage = 'https://mxnet.io/'
+description = """Flexible and Efficient Library for Deep Learning"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'cstd': 'c++14', 'opt': True, 'pic': True, 'openmp': True, 'extra_cflags': '-lflexiblas'}
+
+# MXNet pulls in a bunch of submodules with specific commits
```

Comment:
add comment to mention source for these commit IDs: https://github.com/apache/mxnet/tree/1.9.1/3rdparty